GooglePlayInstant/Editor/BuildSettingsWindow.cs

```diff
@@ -35,6 +35,7 @@ public class BuildSettingsWindow : EditorWindow
 #else
             new LegacyAndroidManifestUpdater();
 #endif
+        private static BuildSettingsWindow _windowInstance;
```

Comment:
Is there a method like Destroy() or something where we should set `_windowInstance = null;` ?

Reply:
Yes. Thanks. 

Added an implementation of OnDestroy() which is now going to set _windowInstance to null to free the reference.